types/amplitude-js/index.d.ts

```diff
@@ -202,3 +205,7 @@ export function logEventWithTimestamp(event: string, data?: any, timestamp?: num
 export function getInstance(instanceName?: string): AmplitudeClient;
 export const __VERSION__: string;
 export const options: Config;
+
+declare const defaultInstance: AmplitudeClient;
+
+export default defaultInstance;
```

Comment:
What changed here?  Did the underlying library gain a default export?

Reply:
Actually it always had one: https://github.com/amplitude/Amplitude-JavaScript/blob/master/src/index.js
 * I'm not sure why/how it worked without this code
 * I got confused by a reviewer bot mentioning
```
The declaration doesn't match the JavaScript module 'amplitude-js'. Reason:
The JavaScript module can be called or constructed, but the declaration module cannot.
```
and I thought this is why my PR wasn't reviewed.

I'm OK to drop it as my usecase works without it and I'm OK to keep it as well as it's closer to the library actual definition.

Reply:
LGTM.  Thanks for confirming!

Reply:
Oh, I see, the comment from the danger bot.  Let me look into that.

Reply:
It looks like the library is published in several forms: https://unpkg.com/browse/amplitude-js@7.2.2/

Personally, I'd recommend just not touching the exports, if you don't need them.  The rest of your change looks straightforward, so there's no reason to open that can of worms.

Reply:
Alright, I've dropped the export change.